### PR TITLE
Merge web tooling changes into main feature branch for TF-571

### PR DESCRIPTION
**Pull request by cartmouse**
- [ ] Code Review
- [ ] Developer Testing

### CODE REVIEW

File: TF_Tooling_Web/src/Connection/ServiceConnection.ts [hunk 1]
```diff
@@ -268,4 +276,24 @@ export class ServiceConnection {
 
         this.webSocket.send(message);
     }
+
+    // Function: RequestTrackingState
+    // Requests 
+    RequestTrackingState(_callback: (detail: TrackingStateResponse) => void) {
+        if (!_callback) {
+            console.error('Request failed. This is due to a missing callback');
```
Comment (NatGreenwood-UL) [inline TF_Tooling_Web/src/Connection/ServiceConnection.ts]: We've got quite a few of these now, wonder if we should be improving this message to clarify which call causes the issue (especially as the function doesn't _actually_ error)

**Review by NatGreenwood-UL — COMMENTED** [review]
Just one minor comment, could you also merge the update to the base branch into this so I can test with the Tooling Test App repo?

File: TF_Tooling_Web/src/Connection/ServiceConnection.ts (hunk 1 above)
Comment (cartmouse) [reply TF_Tooling_Web/src/Connection/ServiceConnection.ts]: A good shout, I'll make these error messages a bit more specific to the calls they're making.

**Review by NatGreenwood-UL — APPROVED** [review]
I've now used this in the TF Test Application to check that the flow through both this, service and onto Tracking system, which all worked great! Happy for this to be merged into the main 571 branch now, I'll get the Unity Tooling done there also

**Pull request merged**